index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portafolio</title>
+</head>
+<body>
+    <nav class="nav">
+        <header class="logo">My Logo</header>
+        <div class="hamburger">
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+        </div>
+    </nav>
+    <section class="content">
+        <h1 class="headline">I’m Mariane Glad to see you!</h1>
+        <p class="description">
+            I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <h2 class="social">LET’S CONNECT</h2>
+        <section class="social-icon">
+            <img class="icon" src="img\Vector.svg" alt="Tweeter icon">
+            <img class="icon" src="img\Linkedin icon.svg" alt="Linkedin icon">
+            <img class="icon" src="img\Vector m.svg" alt="M icon">
+            <img class="icon" src="img\github icon.svg" alt="github icon">
+            <img class="icon" src="img\peace icon.svg" alt="peace icon">
+        </section>
```

Comment:
Kindly use a ul tag to group the list of icons, and li per icon; each icon should also have a link (a tag).
Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href. :+1: